sidebars.js

```diff
@@ -828,12 +827,13 @@ module.exports = {
             ),
           ],
         },
-        require("./docs/apis-tools/tasklist-api-rest/sidebar-schema"),
         require("./docs/apis-tools/web-modeler-api/sidebar-schema"),
         require("./docs/apis-tools/zeebe-api/sidebar-schema"),
         {
           Deprecated: [
+            require("./docs/apis-tools/operate-api/sidebar-schema"),
             require("./docs/apis-tools/tasklist-api/sidebar-schema"),
```

Comment:
@akeller Is this tasklist GraphQl? If yes, this needs to be removed from the `deprecated` section.

Reply:
Will do!

Reply:
@aleksander-dytko, I'm running into a lot of link issues. Let's do this in a separate PR.

Reply:
https://github.com/camunda/camunda-docs/pull/5080